containers/init-image/init-image.changes.releng.bump

```diff
@@ -0,0 +1,2 @@
+- Bump version to 5.1.0
+- Switch to release-stage beta and tag as 5.1 alpha1
```

Comment:
Not really that relevant for Uyuni/SUSE Manager users/customers, so I'd not add it. Besides as this will be part of Uyuni, it can confuse some people. Same for the other containers. I'd remove such entries.